Dockerfile

```diff
@@ -0,0 +1,43 @@
+# python 3
+FROM ubuntu:focal
+
+MAINTAINER K. Shankari (shankari@eecs.berkeley.edu)
+
+WORKDIR /usr/src/app
+
+RUN apt-get update
+RUN apt-get install -y curl
+RUN apt-get install -y git
+
+# install nano and vim for editing
+RUN apt-get -y install nano vim
+
+# cleanup
+RUN apt-get -y remove --purge build-essential
+RUN apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+COPY . .
+
+# ARG SERVER_REPO
+# ENV SERVER_REPO=${SERVER_REPO:-https://github.com/e-mission/e-mission-server.git}
+
+# ARG SERVER_BRANCH
+# ENV SERVER_BRANCH=${SERVER_BRANCH:-master}
+
+# ADD clone_server.sh /clone_server.sh
+RUN chmod u+x ./.docker/setup_config.sh
+# ADD index.html /index.html
```

Comment:
Why remove this? Without this, there will be no `/index.html` to copy 
https://github.com/e-mission/e-mission-server/pull/875/files#diff-53eb57ac0b49c269c7061b4c454813b0214d14cfbdd45f6950e9367698891eb4R8

Reply:
Right. `index.html` doesn't exist in the `e-mission-server` repo. In `e-mission-docker`, it's just a simple html file that says `e-mission server running successfully!` Without it, I see the nice dashboard page when things work.

I think I commented the line out early in this task as an easy way to make a problem go away, and forgot about it. Never thought back to if we wanted a better fix since the end result was still nice.

Reply:
The problem with having a "nice looking" webapp is that it bloats the server code and adds a bunch of dependencies that we then need to keep up to date, or worry about vulnerabilities.

We now have study-specific join pages and an NREL-hosted version of the documentation.

I've actually removed the webapp directory completely. 
https://github.com/e-mission/e-mission-server/commit/df715ee265db00abc8489a63a9912042380bb3f7

Actually, given that, maybe we don't need the index.html, given that 
`if [ -d "webapp/www/" ]; then` will always be false

@aGuttman do you still have a `webapp` directory? Have you pulled from the server recently?

Reply:
I don't have a `webapp` dir in `e-mission-server`. I'm pretty sure that I was using the latest version of `gas-based-detection` too when I created this image that's being used for the cronjob test.

Where *is* the webpage coming from?

Reply:
ok, so do you have an image not on the `ceo_ebike_project` branch?
If so, you can use `docker run` to create a container from it and confirm that there is no `webapp`, no `index.html`
Not sure what it does by default

If not, I will merge this for now (since it is not super important) and you can clean this up in the next round when you have an actual image to experiment with.

Reply:
Yes, I have an image for the `gis` branch that I made yesterday. No `webapp`, no `index.html`. It redirects to https://www.nrel.gov/transportation/openpath.html when you point a browser at it.

Reply:
> It redirects to https://www.nrel.gov/transportation/openpath.html when you point a browser at it.

that's what I thought I did, but couldn't find it in the code

```
@route('/')
def index():
  return static_file("index.html", static_path)
```

Ah I have all 404 redirect to the openpath page

```
@error(404)
def error404(error):
    response.status = 301
    response.set_header('Location', "https://www.nrel.gov/transportation/openpath.html")
```

Ok I am fine with merging this now, and as part of the cleanup task, we can decide whether we want to reintroduce a simple static file. Can you add that as a task to the cleanup issue?